Sources/ExpandedTabBar/ExpandedTabBarViews.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ExpandedTabBarViews.swift
+//  ExpandedTabBar
+//
+//  Created by Yervand Saribekyan on 3/13/18.
+//  Copyright © 2018 Yervand Saribekyan. All rights reserved.
+//
+#if !os(macOS)
+
+import UIKit
+
+internal final class ExpandedTabBarViews {
+
+    class func moreItemView(for item: UITabBarItem,
```

Comment:
Why to not use `static` here as you can't override it due to `internal` and `final` modifiers.

Reply:
Good point)